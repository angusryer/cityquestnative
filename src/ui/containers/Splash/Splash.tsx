import React from "react";
import { Button, Text, View } from "react-native";
import styles from "./Splash.styles";
import { useContext } from "react";
import { PlayerContext } from "../../../context/playerContext";

export default function Splash() {
	// const { playerName, dispatch } = useContext(PlayerContext);
	// const createNewPlayer = () => {
	// 	console.log(playerName);
	// 	dispatch({ type: "new_player", payload: "BOB" })
	// };
	// const deletePlayer = () => {
	// 	dispatch({ type: "delete_player", payload: "BOB" })
	// };

	return (
		<View style={styles.container}>
			<Text>Test</Text>
			{/* <Button title='New Player' onPress={createNewPlayer}>
				New player!
			</Button>
			<Button title='Delete Player' onPress={deletePlayer}>
				Delete player!
			</Button> */}
		</View>
	);
}
