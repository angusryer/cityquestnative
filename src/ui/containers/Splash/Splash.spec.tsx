import React from "react";
import renderer from "react-test-renderer";
import Splash from "./Splash";

describe("Splash screen", () => {
    it("<Splash /> matches previous snapshot", () => {
        const tree: any = renderer.create(<Splash />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})