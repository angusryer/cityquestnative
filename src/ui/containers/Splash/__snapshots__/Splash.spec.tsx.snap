// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash screen <Splash /> matches previous snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <Text>
    This is the Splash Screen
  </Text>
</View>
`;
